index.html

```diff
@@ -16,84 +16,122 @@
 </head>
 
 <body>
+<div class="container">
```

Comment:
Remember the container, row, and sizing order so that the container stretches over the entire page.